src/lib.rs

```diff
@@ -90,14 +92,42 @@ pub struct Ammonia<'a> {
 impl<'a> Default for Ammonia<'a> {
     fn default() -> Self {
         let tags = hashset![
-            "a", "b", "blockquote", "br", "code", "dd", "del", "dl", "dt",
-            "em", "i", "h1", "h2", "h3", "hr", "img", "kbd", "li", "ol", "p",
-            "pre", "s", "strike", "strong", "sub", "sup", "table", "tbody",
-            "td", "th", "thead", "tr", "ul", "hr"
-        ];
-        let generic_attributes = hashset![
-            "title"
+            "a",
```

Comment:
I don't really like this, I think it takes way too much space.

Reply:
Neither do I, but I'm not sure it's worth fighting `rustfmt`.

Reply:
`#[cfg_attr(rustfmt, rustfmt_skip)]` seems easy enough to me.

Reply:
Well, that was easier than I thought 😄.